src/test/groovy/com/gooddata/md/visualization/VisualizationObjectTest.groovy

```diff
@@ -168,12 +172,13 @@ class VisualizationObjectTest extends Specification {
 
         where:
         attributes                                | expected
-        segmentedLineChart.getAttributes()        | [ segmentedLineChart.content.buckets.get(0).getItems().get(0),
-                                                      segmentedLineChart.content.buckets.get(1).getItems().get(0) ]
+        segmentedLineChart.getAttributes()        | [ segmentedLineChart.buckets.get(0).getItems().get(0),
```

Comment:
would it be possible to use just `grrovy array notation` instead of `get`?

Reply:
fixed